sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -80,6 +81,18 @@
     @SuppressLint("StaticFieldLeak")
     private static Analytics sInstance;
 
+    /**
+     * Transmission interval minimal value.
+     */
+    @VisibleForTesting
+    static final int TRANSMISSION_INTERVAL_MINIMUM = 3;
```

Comment:
The convention is to suffix with `IN_SECONDS`.

TRANSMISSION_INTERVAL_MINIMUM -> MINIMUM_TRANSMISSION_INTERVAL_IN_SECONDS